data/virt_autotest/guest_params_xml_files/sles_15_sp3_64_kvm_hvm_sev_es_x86_64.xml

```diff
@@ -0,0 +1,92 @@
+    <guest_installation_method>location</guest_installation_method>
+    <guest_installation_extra_args>sshd=1#sshpassword=nots3cr3t#console=ttyS0,115200n8#textmode=1#rd.debug#rd.memdebug=4#rd.udev.debug</guest_installation_extra_args>
+    <guest_installation_wait></guest_installation_wait>
+    <guest_installation_method_others></guest_installation_method_others>
+    <guest_installation_media>http://openqa.suse.de/assets/repo/fixed/SLE-15-SP3-Full-x86_64-GM-Media1/</guest_installation_media>
+    <guest_installation_fine_grained></guest_installation_fine_grained>
+    <guest_boot_settings>loader=/usr/share/qemu/ovmf-x86_64-code.bin,loader.readonly=yes,loader.type=pflash,loader.secure=no,nvram.template=/usr/share/qemu/ovmf-x86_64-vars.bin</guest_boot_settings>
+    <guest_secure_boot>false</guest_secure_boot>
+    <guest_os_variant></guest_os_variant>
+    <guest_storage_path></guest_storage_path>
+    <guest_storage_type>disk</guest_storage_type>
+    <guest_storage_format>qcow2</guest_storage_format>
+    <guest_storage_label>gpt</guest_storage_label>
+    <guest_storage_size></guest_storage_size>
+    <guest_storage_others></guest_storage_others>
+    <guest_network_type>bridge</guest_network_type>
+    <guest_network_device>br123</guest_network_device>
+    <guest_network_others>model=virtio,rom.bar=off</guest_network_others>
+    <guest_netaddr>192.168.123.0/24</guest_netaddr>
+    <guest_ipaddr></guest_ipaddr>
+    <guest_ipaddr_static>false</guest_ipaddr_static>
+    <guest_macaddr></guest_macaddr> 
+    <guest_graphics>vnc</guest_graphics>
+    <guest_controller></guest_controller>
+    <guest_input></guest_input>
+    <guest_serial>pty</guest_serial>
+    <guest_parallel></guest_parallel>
+    <guest_channel></guest_channel>
+    <guest_console></guest_console>
+    <guest_hostdev></guest_hostdev>
+    <guest_filesystem></guest_filesystem>
+    <guest_sound></guest_sound>
+    <guest_watchdog></guest_watchdog>
+    <guest_video>cirrus</guest_video>
+    <guest_smartcard></guest_smartcard>
+    <guest_redirdev></guest_redirdev>
+    <guest_memballoon></guest_memballoon>
+    <guest_tpm></guest_tpm>
+    <guest_rng></guest_rng>
+    <guest_panic></guest_panic>
+    <guest_memdev></guest_memdev>
+    <guest_vsock></guest_vsock>
+    <guest_iommu></guest_iommu>
+    <guest_iothreads></guest_iothreads>
+    <guest_seclabel></guest_seclabel>
+    <guest_keywrap></guest_keywrap>
+    <guest_cputune></guest_cputune>
+    <guest_memtune></guest_memtune>
+    <guest_blkiotune></guest_blkiotune>
+    <guest_memorybacking>locked=yes</guest_memorybacking>
+    <guest_features></guest_features>
+    <guest_clock></guest_clock>
+    <guest_power_management></guest_power_management>
+    <guest_events></guest_events>
+    <guest_resource></guest_resource>
+    <guest_sysinfo></guest_sysinfo>
+    <guest_qemu_command></guest_qemu_command>
+    <guest_launchsecurity>sev,policy=0x07</guest_launchsecurity>
```

Comment:
Would you please explain why policy 0x07 is chosen? According to https://libvirt.org/formatdomain.html#launch-security and https://documentation.suse.com/sles/15-SP3/html/SLES-amd-sev/article-amd-sev.html, multiple values are supported. Will it be helpful to set bit 0 to 0, so that guest debug is possible?